src/components/site/SiteStatus.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const SiteStatus = props => <p>
```

Comment:
weird formatting here. when you get to this, please let me know and we will try and get the formatting sorted out.